release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -132,6 +132,20 @@ The administrator may review the referenced materials and choose to perform the
 
 For more information, see xref:../updating/understanding-upgrade-channels-release.adoc#conditional-updates-overview_understanding-upgrade-channels-releases[Conditional updates] and xref:../updating/updating-cluster-cli.adoc#update-conditional-upgrade-pathupdating-cluster-cli[Updating along a conditional upgrade path].
 
+[id="ocp-4-10-installing-cluster-on-openstack-ovs-dpdk"]
+==== Installing a cluster on {rh-openstack} that uses OVS-DPDK
+
+You can now install a cluster on {rh-openstack-first} for which compute machines run on Open vSwitch with the Data Plane Development Kit (OVS-DPDK) networks. 
+
+// TODO: Link
+
+[id="ocp-4-10-installing-cluster-on-openstack-compute-machine-affinity"]
+==== Setting compute machine affinity during installation on {rh-openstack}
+
+You can now select compute machine affinity when you install a cluster on {rh-openstack}. By default, compute machines are deployed with a soft anti-affinity policy, but you can also choose strict anti-affinity or anti-affinity server policies.
```

Comment:
strict anti-affinity and anti-affinity are redundant, did you mean to mention the affinity counterpart?
```suggestion
You can now select compute machine affinity when you install a cluster on {rh-openstack}. By default, compute machines are deployed with a soft anti-affinity policy, but you can also choose strict anti-affinity, affinity or soft affinity server policies.
```

Reply:
AFAIK, `affinity` is discouraged, so your options are:
* soft anti-affinity (default)
* anti-affinity
* soft affinity 

Is this correct? @mandre @pierreprinetti